File: android/app/src/main/java/com/collectrndemo/simple/modules/SharedConfig.java
package com.collectrndemo.simple.modules;

import java.util.HashMap;
import java.util.Map;

public class SharedConfig {

    public static final String VAULT_ID = "<VAULT_ID>";

    public static final String ENVIRONMENT = "<ENVIRONMENT>";

    public static final String CARD_NUMBER_FIELD_NAME = "card_number";
    public static final String EXPIRATION_DATE_FIELD_NAME = "card_expirationDate";
    public static final String CARD_NUMBER_CONTENT_PATH = "json.payment_card_number";
    public static final String EXPIRATION_DATE_CONTENT_PATH = "json.payment_card_expirationDate";
    public static final String CARD_NUMBER_PAYLOAD_KEY = "payment_card_number";
    public static final String EXPIRATION_DATE_PAYLOAD_KEY = "payment_card_expirationDate";

    public static Map<String, String> showPayload = new HashMap<>();
}

File: android/app/src/main/kotlin/com/collectrndemo/simple/modules/SharedConfig.kt
package com.collectrndemo.simple.modules

object SharedConfig {

    const val VAULT_ID: String = "tntgsbvmics"

    const val ENVIRONMENT: String = "sandbox"

    const val CARD_NUMBER_FIELD_NAME: String = "cardNumber"
    const val EXPIRATION_DATE_FIELD_NAME: String = "expDate"
    const val CARD_NUMBER_CONTENT_PATH: String = "json.payment_card_number"
    const val EXPIRATION_DATE_CONTENT_PATH: String = "json.payment_card_expiration_date"
    const val CARD_NUMBER_PAYLOAD_KEY: String = "payment_card_number"
    const val EXPIRATION_DATE_PAYLOAD_KEY: String = "payment_card_expiration_date"

    val showPayload: MutableMap<String, String> = HashMap()
}